weekly/2024/week-17.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Week 17 in Packit
+date: 2024-04-29
+authors: lbarczio
+tags:
+  - 2024-April
+  - 2024
+  - April
+---
+
+## Week 17 (April 23rd – April 29th)
+
+- We have fixed the syncing of ACLs during CentOS Stream release syncing.
+  ([packit#2298](https://github.com/packit/packit/pull/2298))
+- `koji_build` job has a new `sidetag_group` option that allows to perform a downstream Koji build in a sidetag.
+  A new sidetag will be created for each configured `dist_git_branch` if it doesn't already exist.
+  ([packit-service#2409](https://github.com/packit/packit-service/pull/2409))
```

Comment:
@nforro should I leave it here or you want to test this more first?

Reply:
I think you can leave it here, but I would probably add something like:

```suggestion
- `koji_build` job has a new `sidetag_group` option that allows to perform a downstream Koji build in a sidetag.
  A new sidetag will be created for each configured `dist_git_branch` if it doesn't already exist.
  This is not very useful right now, but it is the first step on the way to multi-package Bodhi updates, stay tuned.
  ([packit-service#2409](https://github.com/packit/packit-service/pull/2409))
```